airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -453,7 +453,7 @@ def _make_safe_pod_id(safe_dag_id, safe_task_id, safe_uuid):
         :param safe_task_id: a task_id with only alphanumeric characters
         :param random_uuid: a uuid
         :return: ``str`` valid Pod name of appropriate length
-        """
+        """  # noqa: W605
```

Comment:
Because the docstring isn't a rawstring, the escape sequence in the example regex triggers the W605 flake8 warning: `airflow/contrib/executors/kubernetes_executor.py:456:-279: W605 invalid escape sequence '\.'`.